store/orm/orm.go

```diff
@@ -518,3 +558,203 @@ func underlyingBucketType(bucket interface{}) reflect.Type {
+func (orm *ORM) JobRunsCount() (int, error) {
+	return orm.Count(&models.JobRun{})
+}
+
+// JobRunsSorted returns job runs ordered and filtered by the passed params.
+func (orm *ORM) JobRunsSorted(order SortType, offset int, limit int) ([]models.JobRun, int, error) {
+	count, err := orm.JobRunsCount()
+	if err != nil {
+		return nil, 0, err
+	}
+
+	query := orm.Select().OrderBy("CreatedAt").Limit(limit).Skip(offset)
+	if order == Descending {
+		query = query.Reverse()
+	}
+
+	var runs []models.JobRun
+	err = query.Find(&runs)
+	if err == storm.ErrNotFound {
+		err = nil
+	}
+	return runs, count, err
+}
+
+// JobRunsSortedFor returns job runs for a specific job spec ordered and
+// filtered by the passed params.
+func (orm *ORM) JobRunsSortedFor(id string, order SortType, offset int, limit int) ([]models.JobRun, int, error) {
+	count, err := orm.JobRunsCountFor(id)
+	if err != nil {
+		return nil, 0, err
+	}
+
+	query := orm.Select(q.Eq("JobID", id)).OrderBy("CreatedAt").Limit(limit).Skip(offset)
+	if order == Descending {
+		query = query.Reverse()
+	}
+
+	var runs []models.JobRun
+	err = query.Find(&runs)
+	if err == storm.ErrNotFound {
+		err = nil
+	}
+	return runs, count, err
+}
+
+// BridgeTypes returns bridge types ordered by name filtered limited by the
+// passed params.
+func (orm *ORM) BridgeTypes(offset int, limit int) ([]models.BridgeType, int, error) {
+	count, err := orm.Count(&models.BridgeType{})
+	if err != nil {
+		return nil, 0, err
+	}
+
+	var bridges []models.BridgeType
+	err = orm.AllByIndex("Name", &bridges, storm.Skip(offset), storm.Limit(limit))
+	return bridges, count, err
+}
+
+// SaveUser saves the user.
+func (orm *ORM) SaveUser(user *models.User) error {
+	return orm.DB.Save(user)
+}
+
+// SaveSession saves the session.
+func (orm *ORM) SaveSession(session *models.Session) error {
+	return orm.DB.Save(session)
+}
+
+// SaveBridgeType saves the bridge type.
+func (orm *ORM) SaveBridgeType(bt *models.BridgeType) error {
+	return orm.DB.Save(bt)
+}
+
+// SaveTx saves the transaction.
+func (orm *ORM) SaveTx(tx *models.Tx) error {
+	return orm.DB.Save(tx)
+}
+
+// SaveInitiator saves the initiator.
+func (orm *ORM) SaveInitiator(initr *models.Initiator) error {
+	return orm.DB.Save(initr)
+}
+
+// SaveHead saves the indexable block number related to head tracker.
+func (orm *ORM) SaveHead(n *models.IndexableBlockNumber) error {
+	return orm.DB.Save(n)
+}
+
+// Save operation that panics to enforce the use of model specific saves.
+func (orm *ORM) Save(data interface{}) error {
+	logger.Panic("Direct saves are not allowed, use orm's model specific save")
+	return nil
+}
+
+// LastHead returns the last ordered IndexableBlockNumber.
+func (orm *ORM) LastHead() (*models.IndexableBlockNumber, error) {
+	numbers := []models.IndexableBlockNumber{}
+	err := orm.Select().OrderBy("Digits", "Number").Limit(1).Reverse().Find(&numbers)
+	if err == storm.ErrNotFound {
+		return nil, nil
+	}
+	if err == nil {
```

Comment:
[Indent error flow.](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow)

Reply:
I'll fix this in a subsequent PR.